contribs/gmf/src/authentication/service.js

```diff
@@ -0,0 +1,242 @@
+goog.provide('gmf.authentication.service');
```

Comment:
File name must be wrote with a uppercase s : Service.js. Cause it's a java-script object.
(Unlike the component, that is a description. The controller must also have an uppercase but anyway it's always private in it's own file)

Also. provide / require must use this uppercase S.
And also the Object name itself